platform/darwin/docs/theme/assets/css/jazzy.css.scss

```diff
@@ -367,6 +367,17 @@ pre code {
   color: $navigation_task_color;
 }
 
+.nav-group-task[data-url^="Protocols/"],
+.nav-group-task[data-name="MGLForegroundStyleLayer"],
+.nav-group-task[data-name="MGLMultiPoint"],
+.nav-group-task[data-name="MGLShape"],
+.nav-group-task[data-name="MGLSource"],
+.nav-group-task[data-name="MGLStyleLayer"],
+.nav-group-task[data-name="MGLTileSource"],
+.nav-group-task[data-name="MGLVectorStyleLayer"] {
```

Comment:
I’d like that very much. What do you think jazzy should expose as an option, given that the idea of an “abstract class” is very much a user-defined concept, not something baked into the language? Perhaps jazzy could allow you to attach arbitrary properties to items in the `custom_categories` configuration variable?